daemon/export_api_system_secureboot_test.go

```diff
@@ -0,0 +1,52 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2024 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package daemon
+
+import (
+	"github.com/snapcore/snapd/overlord/fdestate"
+	"github.com/snapcore/snapd/overlord/state"
+	"github.com/snapcore/snapd/testutil"
+)
+
+type SecurebootRequest = securebootRequest
+
+func MockFdestateEFISecureBootDBUpdatePrepare(
+	f func(st *state.State, db fdestate.EFISecurebooKeystDB, payload []byte) error,
+) (restore func()) {
+	restore = testutil.Backup(&fdestateEFISecureBootDBUpdatePrepare)
+	fdestateEFISecureBootDBUpdatePrepare = f
+	return restore
```

Comment:
```suggestion
	return testutil.Mock(&fdestateEFISecureBootDBUpdatePrepare, f)
```

Reply:
I wanted to do that, but the branch hasn't been rebased on master yet to include testutil.Mock()

Reply:
right, I didn't notice that